modules/jaggery-apps/app-publisher-web/apis/v1/ssoconfig_api_router.jag

```diff
@@ -82,23 +82,20 @@ require('/modules/publisher.js').exec(function(ctx) {
         var webApp = new WebAppModel(appId);
         webApp.setAcsURL(params.app_acsURL);
         var ssoProvider = new SSOProvider();
-        var app_provider = params.app_provider;
-        var result = null;
         var store = jagg.module("manager").getAPIStoreObj();
-        var isEmailLoginEnabled = store.isEnableEmailUsername();
 
-        if (!isEmailLoginEnabled && app_provider != null && app_provider.indexOf("-AT-") != -1) {
-            result = app_provider.split("-AT-");
-        }
+        var tenantDomain = Packages.org.wso2.carbon.context.CarbonContext.getThreadLocalCarbonContext().getTenantDomain();
```

Comment:
I think this should be fine since we are only retrieving the tenant domain and tenantId. PrivilegedCarbonContext should have used if we calling setter methods of these operations. 
https://docs.wso2.com/display/Carbon420/CarbonContext+API